zookeeper-server/src/test/java/org/apache/zookeeper/server/NettyServerCnxnTest.java

```diff
@@ -184,4 +191,135 @@ public void run() {
             assertArrayEquals("unexpected data", "test".getBytes(StandardCharsets.UTF_8), contents);
         }
     }
+
+    @Test
+    public void testEnableDisableThrottling_secure_random() throws Exception {
```

Comment:
Perhaps I've asked this before, but why don't you use junit's parameterized tests?

Reply:
Yes, you mentioned that in a separate PR before :)

Parameterized test will run the combination for every tests in the class, which makes sense if it's a class wide test which is for testing classes implementing the same interface. Here it's just a test combination for the specific test scenario with enable/disable throttling, so that's why I think using a help method is better. Let me know if this makes sense or not.

Reply:
Makes sense. You would have to create a separate class for it.